drivers/gpu/drm/vc4/vc4_plane.c

```diff
@@ -479,7 +479,11 @@ static u32 vc4_lbm_size(struct drm_plane_state *state)
 		lbm = pix_per_line * 16;
 	}
 
-	lbm = roundup(lbm, 32);
+	/* Align it to 64 or 128 (hvs5) bytes */
+	lbm = roundup(lbm, vc4->hvs->hvs5 ? 128 : 64);
```

Comment:
`error: ‘vc4’ undeclared`

Reply:
Ah - left out when ignoring debug - added back now.

Reply:
But not pushed?

Reply:
Pushed now.